### PR TITLE
ansiweather: add German translation

### PR DESCRIPTION
**Pull request by micschwarz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/ansiweather.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ansiweather
+
+> Ein Shell-Skript um die aktuellen Wetterbedingungen in einem Terminal anzuzeigen.
+> Mehr Informationen: <https://github.com/fcambus/ansiweather>.
```
Comment (kbdharun) [inline pages.de/common/ansiweather.md:4]:
```suggestion
> Weitere Informationen: <https://github.com/fcambus/ansiweather>.
```

**Review by kbdharun — APPROVED** [review]

File: pages.de/common/ansiweather.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# ansiweather
+
+> Ein Shell-Skript um die aktuellen Wetterbedingungen in einem Terminal anzuzeigen.
+> Weitere Informationen: <https://github.com/fcambus/ansiweather>.
+
+- Zeigt eine Vorhersage für die nächsten fünf Tage für Rzeszow, Polen in metrischen Einheiten an:
```
Comment (marchersimon) [inline pages.de/common/ansiweather.md:6]: We write our descriptions in imperative mood:
```suggestion
- Zeige eine Vorhersage für die nächsten fünf Tage für Rzeszow, Polen in metrischen Einheiten an:
```

File: pages.de/common/ansiweather.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`ansiweather -u {{metric}} -f {{5}} -l {{Rzeszow,PL}}`
+
+- Zeigt eine Vorhersage mit Symbolen und Tageslichtdaten für den aktuellen Standort an:
```
Comment (marchersimon) [inline pages.de/common/ansiweather.md:10]:
```suggestion
- Zeige eine Vorhersage mit Symbolen und Tageslichtdaten für den aktuellen Standort an:
```

File: pages.de/common/ansiweather.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`ansiweather -s {{true}} -d {{true}}`
+
+- Zeigt eine Vorhersage mit Wind- und Luftfeuchtigkeitsdaten für den aktuellen Standort an:
```
Comment (marchersimon) [inline pages.de/common/ansiweather.md:14]:
```suggestion
- Zeige eine Vorhersage mit Wind- und Luftfeuchtigkeitsdaten für den aktuellen Standort an:
```

**Review by EmilySeville7cfg — APPROVED** [review]
No fixes found, approved.
Thanks for a contribution! ❤️

**Pull request merged**